bundles/org.eclipse.swt/Eclipse SWT PI/common/org/eclipse/swt/internal/Library.java

```diff
@@ -13,15 +13,10 @@
  *******************************************************************************/
 package org.eclipse.swt.internal;
 
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.net.JarURLConnection;
-import java.net.URL;
-import java.net.URLConnection;
-import java.nio.file.Files;
-import java.nio.file.StandardCopyOption;
-import java.util.jar.Attributes;
+import java.io.*;
+import java.net.*;
+import java.nio.file.*;
+import java.util.jar.*;
 
```

Comment:
Undo above change, in SWT we usually to specific imports:
- To keep things straightforward and avoid any possible class name collision